runtime/oti/vm_api.h

```diff
@@ -517,6 +517,18 @@ isCRIUSupportEnabled(J9VMThread *currentThread);
 BOOLEAN
 isCRIUSupportEnabled_VM(J9JavaVM *vm);
 
+/**
+ * @brief Queries if CRIU security provider needs to be disabled
+ * when CRIU support is enabled. By default support
+ * is not enabled, it can be enabled with `-XX:-CRIUSecProvider`
+ * right after `-XX:+EnableCRIUSupport`
+ *
+ * @param currentThread vmthread token
+ * @return TRUE if enabled, FALSE otherwise
+ */
```

Comment:
Suggest improved comment:
```c
/**
 * @brief Queries if the CRIU security provider should be disabled when CRIU
 * checkpoints are allowed. By default it is enabled, it can be disabled with
 * -XX:-CRIUSecProvider.
 *
 * @param currentThread vmthread token
 * @return TRUE if disabled, FALSE otherwise
 */
```

Reply:
Updated

Reply:
This documentation is also stale.

Reply:
Updated